expat/CMakeLists.txt

```diff
@@ -130,10 +130,10 @@ if(EXPAT_WITH_LIBBSD)
 endif()
 
 if(MSVC)
-    # Minimum supported MSVC version is 1800 = Visual Studio 12.0/2013
+    # Minimum supported MSVC version is 1910 = Visual Studio 15.0/2017
     # See also https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html
-    if(MSVC_VERSION VERSION_LESS 1800)
-        message(SEND_ERROR "MSVC_VERSION ${MSVC_VERSION} is not a supported Visual Studio compiler version. Please use Visual Studio 2013 or any later version.")
+    if(MSVC_VERSION VERSION_LESS 1910)
+        message(SEND_ERROR "MSVC_VERSION ${MSVC_VERSION} is not a supported Visual Studio compiler version. Please use Visual Studio 15.0/2017 or any later version.")
```

Comment:
@hartwork expat 2.3.0 seems to be perfectly valid for Visual Studio 2015 maybe even 2013, I can understand you don't want to test Visual Studio < 2017, but is there any reason to force an error in CMakeLists?

Reply:
Hi @SpaceIm, I'm happy to both explain and discuss that change in detail. Would you mind dropping that comment of yours into a new issue so that it's easier to see for others that are interested? Doesn't have to be super long. Thanks in advance! :pray:

Reply:
sure: https://github.com/libexpat/libexpat/issues/454

Reply:
Thank you!